static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -142,6 +149,9 @@ function Onboarding(props: Props) {
     );
   };
 
+  if (!stepObj || stepIndex === -1) {
+    return <div>Can't find</div>;
```

Comment:
@Neo-Zhixing what is this supposed to be?

Reply:
Should be impossible to happen, i think it was just there to convince TypeScript